cmd/gce-pd-csi-driver/main.go

```diff
@@ -64,6 +64,9 @@ var (
 	waitForOpBackoffSteps     = flag.Int("wait-op-backoff-steps", 100, "Steps for wait for operation backoff")
 	waitForOpBackoffCap       = flag.Duration("wait-op-backoff-cap", 0, "Cap for wait for operation backoff")
 
+	enableDeviceInUseTimeout = flag.Bool("enable-device-in-use-timeout", true, "If set to true, ignores device in use errors when attempting to unstage a device if it has been stuck for longer than 'device-in-use-timeout'")
```

Comment:
I think the boolean flag should bypass the "device in use" check entirely, and be renamed `enable-device-in-use-check-on-node-unstage`. I don't see much value in having a boolean for using the timeout or not (eg: if a user really wants a disable the timeout, they can set `device-in-use-timeout` to infinity). I think the value of the switch more for users where the device in use check is blocking NodeUnstage due to the way `/sys/fs/` is reacting, and wants to skip in entirely, for all disks.

In retrospect, this is something that I should have added in https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1658, to allow the feature to be turned off.

Reply:
Ah, I see what you mean. Fixed --the `confirmDeviceUnused` in node.go will now always return nil if the flag is true.